core/mapserver/mapserver.go

```diff
@@ -0,0 +1,207 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package mapserver implements the TrillianMapClient interface.
+// To start with, this will use KT's internal utilities to accomplish the same functions.
+// Later this will be replaced with a simple RPC to Trillian Maps.
+package mapserver
+
+import (
+	"crypto"
+	"fmt"
+	"log"
+
+	"github.com/google/keytransparency/core/sequenced"
```

Comment:
Done. 

Quick note that the reason for this separation has changed. 
Previously, we were:
1. Checking out the project into `go/src/github.com/username/keytransparency` 
2. Running `sed` scripts to rewrite all the dependencies locally. 
3. `go fmt` was reshuffling import orderings based on the local username causing unnecessary churn.

Now, we figured out that best practice is slightly different:
1. Fork the project in github under your local username
2. `git clone github.com/username/keytransparency` into `src/github.com/google/keytransparency` 

Now you can develop locally and push to your fork without needing to rewrite and reshuffle the imports. 

Therefore, we could (but we don't have to) remove the requirement to separate KT and non-KT imports in the import block. For consistency, let's continue to split them, this has just been for background info.